djangocms_attributes_field/forms.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
Was following the conventions set in JSONField, but I suppose there's no harm in moving it to fields.py.